dist/idiomorph.js

```diff
@@ -110,7 +110,7 @@ var Idiomorph = (function () {
          * @returns {boolean}
          */
         function ignoreValueOfActiveElement(possibleActiveElement, ctx) {
-            return ctx.ignoreActiveValue && possibleActiveElement === document.activeElement;
+            return ctx.ignoreActiveValue && possibleActiveElement === document.activeElement && possibleActiveElement !== document.body;
```

Comment:
@1cg this line (along with the rest of the generated portion of the diff) might be best merged as part of https://github.com/bigskysoftware/idiomorph/pull/47 (ahead of this PR).